### ISSUE TITLE
Can not open

### ISSUE THREAD

**Issue by jalenchuh** [Genymobile/scrcpy#2101]
- [z] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [x] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Windows
 - scrcpy version: 1.17
 - installation method: choco
 - device model: Meizu 16th
 - Android version: 10

**Describe the bug**

Can not open the windows, but the screen of phone is on

```
INFO: scrcpy 1.17 <https://github.com/Genymobile/scrcpy>
C:\ProgramData\chocolatey\lib\scrcpy\tools\scrcpy-server: 1 file pushed, 0 skipped. 16.1 MB/s (34930 bytes in 0.002s)
[server] INFO: Device: Meizu 16th (Android 10)
Segmentation fault
INFO: Renderer: direct3d
INFO: Initial texture: 1080x2160
WARN: Device disconnected
```

**Comment by rom1v** [comment]
The segmentation fault occurs on the device. Could you try with the official release, please? (instead of chocolatey)

**Comment by jalenchuh** [comment]
I tried to use official release but it still doesn't work.

**Comment by rom1v** [comment]
OK. Could you post the logcat, please?

```
adb logcat -c
scrcpy
# once it closes
adb logcat -d > logcat.txt
```

Then post `logcat.txt`.

**Comment by jalenchuh** [comment]
[logcat.txt](https://github.com/Genymobile/scrcpy/files/5955576/logcat.txt)

**Comment by chenaichenet** [comment]
yeah,I have same problem,I think this problem is caused by Meizu intenal test system.In the intenal test system ,Meizu delete this function.

**Comment by jalenchuh** [comment]
I have asked the engineer of Meizu, they sayaid they will test it.
Flyme 9 is coming, I hope and believe it will be fixed.

**Comment by chenaichenet** [comment]
> I have asked the engineer of Meizu, they sayaid they will test it.
> Flyme 9 is coming, I hope and believe it will be fixed.

好的，希望9能快点出来，不过我看我的16系列有点难，4月底可能还不行。

**Comment by EugeneJie** [comment]
It still doesn't work on Flyme 9 ...
